### ISSUE TITLE
Fungal genome sizes are missing

### ISSUE THREAD

**Issue by rjchallis** [genomehubs/goat-data#45]

**Comment by sujaikumar** [comment]
Thanks for spotting this. Found the source of the error! It is happening because the genome_size attribute in sources/genomesize_karyotype/FILE_www_zbi_ee_fungal_genomesize_20210319.types.yaml had type long and didn't have the right function. I've fixed it now to:
```
  genome_size:
    header: "1C in Mbp"
    function: "{} * 1000000"
    separator:
      - ","
      - "(?<!e)-"
    type: long
```

**Comment by sujaikumar** [comment]
This has been fixed as of 2023.06.26 - thanks!
